__tests__/app.test.js

```diff
@@ -232,4 +233,107 @@ describe("POST /api/reviews/:review_id/comments", () => {
         expect(body.msg).toBe("Invalid datatype found");
       });
   });
+  test("201: responds with the posted comment and ignores addiotnal properties in the comment", () => {
+    const newComment = {
+      username: "mallionaire",
+      body: "I love this game!",
+      email: "Johnparker123@gmail.com",
+      city: "Manchester",
+    };
+    return request(app)
+      .post("/api/reviews/4/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toMatchObject(
+          expect.objectContaining({
+            comment_id: 7,
+            body: "I love this game!",
+            votes: 0,
+            author: "mallionaire",
+            review_id: 4,
+            created_at: expect.any(String),
+          })
+        );
+      });
+  });
+});

```

Comment:
This is a nice test! However, should have been part of another ticket preferably